README.rst

```diff
@@ -14,7 +14,20 @@ The main purpose of this estimator is to give designers an easy way to choose pa
 Quick Start
 -----------
 
-- Usage
+We currently provide evaluators for the security of the `LWE`, `NTRU`, and `SIS` problems. 
+Our estimator integrates simulators for the best known attacks against these problems, and provides
+bit-security estimates relying on heuristics to predict the cost of lattice reduction algorithms, the default being
+using `[MATZOV22] <https://zenodo.org/record/6412487>` for lattice reduction cost, and the GSA assumption 
```

Comment:
We have a [references file](https://github.com/malb/lattice-estimator/blob/main/docs/references.rst). Also, we shouldn't hardcode these choices here but document `conf.py` and point the reader there from here.